src/modules/DecodeQr/Module.js

```diff
@@ -15,19 +14,13 @@ module.exports = function DoNothing(options, UI) {
 
     var step = this;
 
-    getPixels(input.src, function(err, pixels) {
-
-      if (err) throw err;
-
+    function extraManipulation(pixels){
       var w = pixels.shape[0];
       var h = pixels.shape[1];
       var decoded = jsQR(pixels.data, w, h);
-
-
-      // Tell Image Sequencer that this step is complete
-      options.step.qrval = (decoded) ? decoded.data : 'undefined';
-    });
-
+      if (options.step.inBrowser && options.step.ui) options.step.qrval = (decoded) ? decoded.data : 'undefined';
```

Comment:
I think you should append `+=` instead of overwrite.

Reply:
done.